lib/codegen/generator.dart

```diff
@@ -292,7 +292,7 @@ class _DeclarationCode extends ConstExpression {
 
   List<String> get librariesUsed => []
```

Comment:
it might be pushed-down by ddc already (they try hard to remove dynamics from allocation sites automatically if they know what it should be).